core/modules/node/node.block.inc

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ *  NodeBlock  extends Block
```

Comment:
Odd extra whitespace going on here. Remove one space before AND after `NodeBlock`.